### DISCUSSION TITLE
How to use Incremental Indexing via 0.4.0?

### DISCUSSION THREAD

**Discussion by xufengnian** [microsoft/graphrag#1365] [Q&A]
I have used version 0.4.0, the drift mode cost a lot and many models go wrong, but still I got the query results using the gpt-4o model, now my question is, how to use the incremental update feature? Is it directly uploading new files to the input folder or something else?

**Comment by JoedNgangmeni** [comment]
I have a similar question. Can we update the knowledge base without having to rebuild the entire index?

**Comment by AlonsoGuevara** [comment]
Hi! Yes, just add new files into your input folder and then run graphrag update --root your/workspace/path

Make sure storage config in yaml points to an existing index, so graphrag only indexes the new inputs.
